integration/benchmarks_test.go

```diff
@@ -84,18 +84,26 @@ func BenchmarkQuerySmallDocuments(b *testing.B) {
 func BenchmarkReplaceSettingsDocument(b *testing.B) {
 	ctx, collection := setup.Setup(b)
 
+	n := 100
 	iter := shareddata.BenchmarkSettingsDocuments.NewIterator()
-	_, doc, err := iter.Next()
+	docs, err := iterator.ConsumeValuesN(iter, n)
 	iter.Close()
 
 	require.NoError(b, err)
+
+	insertDocs := make([]any, len(docs))
+	for i := range insertDocs {
+		insertDocs[i] = docs[i]
```

Comment:
We could just use a setup like other benchmarks

Reply:
good idea!